index.js

```diff
@@ -219,37 +220,38 @@ function init (ssb, config) {
   /* Tangle: auto-add tangles.group info to all private-group messages */
   const getGroupTangle = GetGroupTangle(ssb, keystore, 'group')
   const getMembersTangle = GetGroupTangle(ssb, keystore, 'members')
-  ssb.publish.hook(function (publish, args) {
-    const [content, cb] = args
-    if (!content.recps) return publish.apply(this, args)
+  // TODO: make this a ssb.tribes.publish function instead of a hook
```

Comment:
`ssb.db.create.hook` I reckon?

Reply:
idk aren't hooks a bit scary? you lose a bit of control of when they actually run